### DISCUSSION TITLE
Doens't show the "Integrations" panel on the User Preferences

### DISCUSSION THREAD

**Discussion by OM-Tecnologia** [codemagic-ci-cd/codemagic-docs#2045] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

https://codemagic.io/settings

This page doens't show the integrations panel, only the "Avatar", "Theme" and "Newsletter"

I was following this guide (the video): https://docs.codemagic.io/flutter-code-signing/ios-code-signing/

have they changed that?

### Expected results

Integrations panel, where i can integrate my iOS account.

### Actual results

I can't integrate the iOS account

### Build id (optional)

6501e37caf3820efb78dba36

**Comment by himesh-cm** [comment, marked as answer]
Hi @OM-Tecnologia, it is under ` Teams > Personal Account > Integrations`.

**Reply by OM-Tecnologia** [reply]
Excelent, thanks.

**Reply by himesh-cm** [reply]
Great. While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
